check_format.py

```diff
@@ -10,9 +11,12 @@
 
 
 def check_format():
+    any_failed = False
     for i in files:
-        system(f"pycodestyle --show-source --show-pep8 --format=default {i}")
+        exit_code = call(["pycodestyle", "--show-source", "--show-pep8", "--format=default", i])
+        any_failed |= True if exit_code == 1 else False
+    return any_failed
 
 
 if __name__ == "__main__":
-    check_format()
+    sys.exit(check_format())
```

Comment:
And we exit with the condition, so that this will fail on GitHub Actions if any code is pushed that is not format compliant.